deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -246,7 +247,8 @@ data:
       @label @NORMAL
     </match>
     <label @NORMAL>
-      #  only match fluentd logs based on fluentd container name
+      #  only match fluentd logs based on fluentd container name. by default, this is
+      #  <filter **collection-sumologic**>
       <filter **collection-sumologic**>
```

Comment:
As I understand we want to change this line to: `{{ printf "<filter **%s**>" (include "sumologic.fullname" .) }}`

Reply:
It's kubernetes, nvm 👍